src/locales/en.yaml

```diff
@@ -0,0 +1,50 @@
+username: Username
```

Comment:
There is a bit of a problem with the format I use here. It needs locale code in the source file for Transifex to parse them, but I will update this as soon as I finish my meeting.